assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js

```diff
@@ -260,23 +260,34 @@ export default function Footer( {
 		setPrevIsOpen( isOpen );
 	}, [ isOpen, prevIsOpen ] );
 
+	const showSelection = ! saveError || saveError?.code;
+
```

Comment:
```suggestion
```

Reply:
I think it might be simpler and more readable to do the following instead:
1. Updating the definition of `saveError` to only return the API error and do nothing in regards to minimum metrics selection.
2. Applying the following (JS) suggestions.